packages/synapse-react-client/src/components/DataGrid/DataGridTypes.ts

```diff
@@ -9,6 +9,18 @@ const gridRowSchema = s.obj({
       versionNumber: s.con(0),
       etag: s.con(''),
     }),
+    rowValidation: s.obj({
+      isValid: s.con(true),
+      validationErrorMessage: s.con(''),
+      allValidationMessages: s.vec(s.con('')),
+      validationException: s.obj({
+        keyword: s.con(''),
+        pointerToViolation: s.con(''),
+        message: s.con(''),
+        schemaLocation: s.con(''),
+        causingExceptions: s.vec(s.con('')),
+      }),
+    }),
```

Comment:
This entire node is actually a constant. Can we do something like this?
```ts
import { ValidationResults } from '@sage-bionetworks/synapse-types'
```


```suggestion
    rowValidation: s.con<ValidationResults | undefined>(undefined),
```